src/Resources/contao/classes/FrontendTemplate.php

```diff
@@ -124,15 +127,11 @@ protected function compile($blnCheckRequest=false)
 			}
 		}
 
-		// Add the output to the cache
-		$this->addToCache();
-
 		// Unset only after the output has been cached (see #7824)
 		unset($_SESSION['LOGIN_ERROR']);
 
-		// Replace insert tags and then re-replace the request_token tag in case a form element has been loaded via insert tag
-		$this->strBuffer = $this->replaceInsertTags($this->strBuffer, false);
-		$this->strBuffer = str_replace(array('{{request_token}}', '[{]', '[}]'), array(REQUEST_TOKEN, '{{', '}}'), $this->strBuffer);
+		// Replace insert tags
+		$this->strBuffer = $this->replaceInsertTags($this->strBuffer);
```

Comment:
Because I want it to be `true` so they are replaced by the `<esi>` tags and not ignored.